src/build_options.rs

```diff
@@ -31,11 +31,12 @@ pub struct BuildOptions {
     ///
     /// This option is ignored on all non-linux platforms
     #[structopt(
-        long,
-        possible_values = &["2010", "2014", "2_12", "2_17", "2_24", "2_27", "off"],
-        case_insensitive = true,
+        name = "compatibility",
+        long = "compatibility",
+        alias = "manylinux",
```

Comment:
Let's keep the `--manylinux` as an alias to `--compatibility` for a while. It'd be nice to somehow emit a warning if `--manylinux` is used instead of `--compatibility`.